sed/loader/flash/loader.py

```diff
@@ -114,11 +105,12 @@ def initialize_paths(self) -> Tuple[List[Path], Path]:
     def get_files_from_run_id(
```

Comment:
Do you create the folders now here and again in the ParquetHandler?

Reply:
yes. let me see how to address this

Reply:
No, this is just looking for h5 files in the raw dir (for the runs passed) and creating a list out of them. Hence, this method has nothing to do with saving the converted parquet files.
I'd have moved this method to BufferHandler but this is part of baseclass so I think it's good to keep it here.